give.php

```diff
@@ -249,6 +252,7 @@ public function init() {
 	private function bindClasses() {
 		$this->container->singleton( 'templates', Templates::class );
 		$this->container->singleton( 'routeForm', FormRoute::class );
+		$this->container->singleton( UncaughtExceptionLogger::class );
```

Comment:
@kjohnson I keep debating with myself as to whether or not I really want this to go through the Service Container. I set this whole thing up before the Service Providers are registered and booted, so we couldn't use DI in this class anyway. It's also meant to be a very simple, standalone class. I'm 80% convinced to just make this a good 'ol `new UncaughtExceptionLogger` case, unless you think otherwise.

Reply:
Yeah, I'm agreeable to separating this out - it is essentially a "bolt on" for isn't intended to be re-used or otherwise integrated.

Reply:
Great! Cleaned up in a42a80b